### PR TITLE
Multiple UI Improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Changes include:
- Improvements to docs page
- Changing the default theme from mattermost to organization
- Adding code theme previews

### CODE REVIEW

File: web/sass-files/sass/partials/_docs.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@charset "UTF-8";
+
+.docks__page {
```
Comment (jwilander) [inline web/sass-files/sass/partials/_docs.scss]: Shouldn't this be 'docs'?

**Comment by jwilander** [comment]
+1, other than naming issue

**Comment by asaadmahmoodspin** [comment]
Done.

File: web/sass-files/sass/partials/_sidebar--right.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
     padding: 0;
     background: #fff;
     @include single-transition(transform, 0.5s, ease);
-    right: -320px;
+    right: -400px;
```
Comment (crspeller) [inline web/sass-files/sass/partials/_sidebar--right.scss]: Was this intentional? It seems random and unrelated.

Comment (asaadmahmoodspin) [inline web/sass-files/sass/partials/_sidebar--right.scss]: Yes intentional, I just checked out that the sidebar width of 400 and the right value I gave was 320 so just fixed that.

**Comment by coreyhulen** [comment]
+1

**Pull request merged**